### DISCUSSION TITLE
Pallet choices

### DISCUSSION THREAD

**Discussion by kenokenobingo** [NucleiStudio/governance-os#37] [General]
I’m wondering why you don’t use the `democracy` and `collective` pallets by the Substrate developers.

**Comment by ETeissonniere** [comment]
Hey Keno, the goal of this project is to provide alternative implementations for projects for which those pallets are not enough.

For instance we support alternative voting systems with new properties such as collusion resistance.

The end goal is to support DAOs on Polkadot, which isn't really possible with the `democracy` or `collective` pallets :).

Does that answer your question?

**Reply by kenokenobingo** [reply]
Yes, thank you! :) Would be nice to have a documentation, explaining both the political and technical implementations.

**Reply by ETeissonniere** [reply]
Indeed, can probably spend time on this. For now it is mostly a one man effort but I wanted to have a website for it at some point. Something pretty and informative at least.
